lib/NodeHelpers.js

```diff
@@ -130,3 +131,44 @@ function getLocalMatrix4(node, result) {
 
     return Matrix4.fromTranslationQuaternionRotationScale(translation, rotation, scale, result);
 }
+
+var packedParametersScratch = {
+    meshes : undefined,
+    parameters : undefined,
+    primitiveFunction : undefined
+};
+
+// Perform an operation on each primitive in a scene.
+// PrimitiveFunction should expect primitive, a meshPrimitiveID, some parameters, and the node itself
+function forEachPrimitiveInScene(gltf, scene, primitiveFunction, parameters) {
```

Comment:
@lilleyse as part of your cleanup roadmap, I suspect we can use this function and friends for the "jQuery for glTF" that we discussed offline.